tests/test_dbstateserver.py

```
@@ -0,0 +1,77 @@
+    return dg
+
+
+@pytest.fixture
+def dbss(ardos):
+    return ardos(md=True, ss=True, db=True, dbss=True)
+
+
+def _seed_player(sender_conn, name="activator") -> int:
+    sender_conn.send(_create_player(name))
+    it = DatagramIterator(sender_conn.recv(timeout=5.0))
+    it.read_header(); it.read_uint32()
```

Ruff (0.15.11)

[error] Multiple statements on one line (semicolon)

(E702)